pyEpiabm/pyEpiabm/core/person.py

```diff
@@ -155,3 +167,21 @@ def remove_place(self, place):
             ind = place_list.index(place)
             self.places.pop(ind)
             self.place_types.pop(ind)
+
+    def close_place(self, closure_place_type):
```

Comment:
Can we rename this method? It sounds like it should actually close a place, rather than checking if they should be closed. Also are places not stored on the microcell level - I think it would make more sense for this to be a property of the microcell rather than the person

Reply:
The closure_start_time is stored in the microcell level and as the household, place and spatial scaling (in foi.py) depend on infector or infectee, so I implement them in person's python file. The strategy is checking if the person is associated with one of the intended close place, if yes, then the household, place and spatial foi related to this person will be scaled. I'll rename the method as 'is_place_closed'?